homeassistant/components/sensor/tibber.py

```diff
@@ -146,8 +148,11 @@ def _update_current_price(self):
             if now.date() == price_time.date():
                 max_price = max(max_price, price_total)
                 min_price = min(min_price, price_total)
+                num += 1
+                avg_price += price_total
```

Comment:
This isn't the average price, but the sum of the prices. I'd rename the variable.